packages/chat-client/package.json

```diff
@@ -31,6 +32,7 @@
     "@walletconnect/core": "2.0.0-rc.4",
     "@walletconnect/jsonrpc-utils": "^1.0.3",
     "@walletconnect/logger": "^1.0.1",
+    "@walletconnect/time": "^1.0.2",
```

Comment:
Adding `@walletconnect/time` explicitly here, which is already present anyways as a sub-dependency of other `@walletconnect/*` pkgs.